src/main/java/appeng/me/storage/AbstractCellInventoryHandler.java

```diff
@@ -122,6 +122,17 @@ public ICellInventory getCellInv()
 
 		return (ICellInventory) ( o instanceof ICellInventory ? o : null );
 	}
+	
+	@Override
+	public void persist()
+	{
+		final ICellInventory cell = this.getCellInv();
+		if (cell != null)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)